applications/citadel.md

```diff
@@ -147,21 +153,16 @@ From our side, we are ready to make the part associated specifically with Polkad
-| 9 | Testing and debugging | Performing different types of testing. Verification of the main scenarios such as sending tokens, and staking in the various environments including Ledger and Trezor wallets. | During this step, we proceed with testing, bug listing on Jira, test-cases writing. As a result, we will have a fully working product with all the crucial bugs fixed. | Alexander (QA Engineer), Aleksey (QA Engineer)40h |
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | The described requirements fully cover the use cases and user interaction with the system.  [ 1. Create and import an address](https://paradigmcitadel.atlassian.net/wiki/external/1741783055/ZWI4NGNjMzgwNTNkNGMxN2EzZjJlOWE0NTQyZjE0MTY)  [2. Integration of staking](https://paradigmcitadel.atlassian.net/wiki/external/1742372869/YjE0ZmE5ZmRmM2VmNDU5NjhkYjViNjVhODZjY2YxMmM?atlOrigin=eyJpIjoiMTBiYWFhMzc3NGIyNGMxNDg1MGYzNzkwYjdmYmI5MTMiLCJwIjoiYyJ9) [3. Integration of transfers](https://paradigmcitadel.atlassian.net/wiki/external/1742209311/ODM1YTc1MGFjNTQ1NGI4Y2FjMDdlOWYyNTg5YmIwZWU?atlOrigin=eyJpIjoiMDRmZDk2ZWFhMjRiNGM1Mzk5NGIzM2IyYzU0YzRjNTkiLCJwIjoiYyJ9) [4. Price and market performance information](https://paradigmcitadel.atlassian.net/wiki/external/1741553682/MGQxZmI1NWIzMmVjNDc4M2ExNDA2OTJiMDk2MWUzNDA?atlOrigin=eyJpIjoiZTU0NzNkMzJhNzI1NGRlZDliNmVlZDY0YzQ5YTRhYjEiLCJwIjoiYyJ9) [5. Integration of transaction ](https://paradigmcitadel.atlassian.net/wiki/external/1745387542/Njk0NWRiNWUyOGY1NDlhZWFjYTYxZDA0OWQ3MzkwZDY?atlOrigin=eyJpIjoiMGNjMzU2NjU1ZGJkNDYzNzk5ZjU3NDVhYWJjMTk2NGIiLCJwIjoiYyJ9)|
+| 0c. | Testing Guide | Each epic describes testing opportunities.  [ 1. Create and import an address](https://paradigmcitadel.atlassian.net/wiki/external/1741783055/ZWI4NGNjMzgwNTNkNGMxN2EzZjJlOWE0NTQyZjE0MTY)  [2. Integration of staking](https://paradigmcitadel.atlassian.net/wiki/external/1742372869/YjE0ZmE5ZmRmM2VmNDU5NjhkYjViNjVhODZjY2YxMmM?atlOrigin=eyJpIjoiMTBiYWFhMzc3NGIyNGMxNDg1MGYzNzkwYjdmYmI5MTMiLCJwIjoiYyJ9) [3. Integration of transfers](https://paradigmcitadel.atlassian.net/wiki/external/1742209311/ODM1YTc1MGFjNTQ1NGI4Y2FjMDdlOWYyNTg5YmIwZWU?atlOrigin=eyJpIjoiMDRmZDk2ZWFhMjRiNGM1Mzk5NGIzM2IyYzU0YzRjNTkiLCJwIjoiYyJ9) [4. Price and market performance information](https://paradigmcitadel.atlassian.net/wiki/external/1741553682/MGQxZmI1NWIzMmVjNDc4M2ExNDA2OTJiMDk2MWUzNDA?atlOrigin=eyJpIjoiZTU0NzNkMzJhNzI1NGRlZDliNmVlZDY0YzQ5YTRhYjEiLCJwIjoiYyJ9) [5. Integration of transaction ](https://paradigmcitadel.atlassian.net/wiki/external/1745387542/Njk0NWRiNWUyOGY1NDlhZWFjYTYxZDA0OWQ3MzkwZDY?atlOrigin=eyJpIjoiMGNjMzU2NjU1ZGJkNDYzNzk5ZjU3NDVhYWJjMTk2NGIiLCJwIjoiYyJ9) |
+| 0d. | Docker | Access will be granted after several iterations |
+| 0e. | Article |  Press release will be prepared a few weeks before the official rollout of the network. The link to the article will be sent when ready
```

Comment:
```suggestion
| 0e. | Article |  Press release will be prepared a few weeks before the official rollout of the network. The link to the article will be sent when ready

| **Number** | **Step** | **Specification** | **Detailed description** | **Team members involved** |
| --- | --- | --- | --- | --- |
| 1 | Node launch (not included in grant amount requirements) | Citadel will join the Polkadot ecosystem as a validator | The Polkadot node will be deployed with the setting of specific glitch recognition notifications (for instance, in case the validator node suspended block production, or server memory faced some issues) via Zabbix and Grafana. As a result of this stage, we expect the node to be up and running with the maintenance system and RPC interface for node access. After the successful node launch, the necessary support and updates will be provided for a longer period. | Nikita (system administrator)120h per year|
| 2 | Design (not included in grant amount requirements) | Interface preparation, design elements specifications, and tuning | As a result of this stage, the mockups for web and mobile UI will be ready on Figma. | Margarita Omelchenko (Lead Designer)40h |
| 3 | Preparation set 1 | Polkadot blockchain scrutiny, data transfer to PostgreSQL (for data aggregation and statistics building) | This stage mainly involves the examination of Polkadot blockchain, parser building for the data transfers to PostgreSQL database (it will allow us to aggregate data conveniently, receive transactions of a certain type on a specific address, etc.)This step will bring us some vital milestones: our database will contain all the transactions - new ones sourced directly from the blockchain will be also stored in our database that is convenient for the development needs (balance/rewards charts creation, etc.)Also, as a result we will possess the following connector methods: (isSystemAddress, validateAddress, getLastBlock, getNextBlock, getOneBlock, filterOperations, getDelegationBalanceInfo). | Yeskendir (Backend Developer), Pavel (Lead Backend Developer)120h |
| 4 | Preparation set 2 | Examination of Polkadot transfer methods and staking system | This step entails the connector methods: prepareTransfer, prepareDelegation, sendTransaction). | Yeskendir (Backend Developer), Pavel (Lead Backend Developer)30h |
| 5 | Transfers integration | Analysis of a network&#39;s transfer transaction; search for necessary libraries or APIs;determination of transaction preparation logic; writing logic; signing; writing the send logic | Upon the successful completion of this step, we will have transactions integrated into web and mobile versions, as well as Backend API methods for the execution of transfers&#39; transactions. | Pavel (Lead Backend Developer), Petr (Lead Frontend Developer), Yeskendir (Backend Developer), Alexander (Frontend Developer), Andrey (Mobile Developer)80 h |
| 6 | Integration of staking | Analysis of network&#39;s staking transaction, search for necessary libraries or APIs;determination of transaction preparation logic; writing logic; signing; writing the send logic. Polkadot validator list parsing. | By the end of this step, we will have staking integrated into web and mobile versions, as well as Backend API methods for the execution of staking transactions via our node. Trezor and Ledger integration for Polkadot. | Pavel (Lead Backend Developer), Petr (Lead Frontend Developer), Yeskendir (Backend Developer), Alexander (Frontend Developer), Andrey (Mobile Developer)80h |
| 7 | Transaction and price feed metrics adding | Finding a suitable price feed solution. Information collection from node&#39;s RPC via API or source libraries. | At this stage, we are preparing the relevant data for charts and the network&#39;s significant metrics. | Yeskendir (Backend Developer)20h |
| 8 | Mobile integration | Development of mobile app for interaction with Polkadot wallets using Web API| For mobile app development (Android &amp; iOS) we will use Flutter SDK. Polkadot wallets will be integrated with C++ trustwallet/wallet-core library. Furthermore, during this step we will work on screen layouts (with previously prepared design) and adaptation for different display resolutions. | Andrey (Mobile Developer)120h |
| 9 | Testing and debugging | Performing different types of testing. Verification of the main scenarios such as sending tokens, and staking in the various environments including Ledger and Trezor wallets. | During this step, we proceed with testing, bug listing on Jira, test-cases writing. As a result, we will have a fully working product with all the crucial bugs fixed. | Alexander (QA Engineer), Aleksey (QA Engineer)40h |
```